src/Sidebar/Sidebar-btn/Sidebar-btn.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './Sidebar-btn.css';
+
+const btn = (props) =>{
+    
+    return (
+        <div className="Sidebar-btn">
+            <span>{props.children}</span>
+            <a href={props.href} className="Sidebar-btn-link">{props.txt}</a>
```

Comment:
Tu mnie trochę zaskoczyłaś :) 
a href przenosi Cię pod określony adres, ale przeładowywuje też stronę. Aktualnie bardzo tego nie widać, ale w pewnym momencie na pewno byś się na to nacięła. Te wszystkie SPA opierają się właśnie na tym, że teoretycznie adres URL się zmienia, ale w praktyce strona nie ulega przeładowaniu. Podejrzewam, że Max opisuje trochę na ten temat w sekcji 11 :) Albo ta dyskusja na Stackoverflow:
https://stackoverflow.com/questions/43087007/react-link-vs-a-tag-and-arrow-function

react router daje coś takiego jak <Link to=..> na miejsce <a href=...>

Reply:
@nettecode Oj, taaaak. Bardzo szybko się nacięłam (robię coś równolegle), ale już po posłaniu do CR.